### ISSUE TITLE
Methods with same name should be ordered based on numbers of parameters

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#253]
Inside a class, methods that have the same name (and accessibility) should be ordered based on numbers of parameters.
